OpenRA.Mods.Common/MapGenerator/MultiBrush.cs

```diff
@@ -28,11 +28,18 @@ namespace OpenRA.Mods.Common.MapGenerator
 	/// </summary>
 	public sealed class MultiBrushInfo
 	{
+		internal static MiniYamlNode SaveField(string key, MiniYaml value) => new(key, value);
+		internal static MiniYamlNode SaveField(string key, string value) => new(key, new MiniYaml(value));
+		internal static MiniYamlNode SaveField(string key, ushort value) => new(key, new MiniYaml(Exts.ToStringInvariant(value)));
+		internal static MiniYamlNode SaveField(string key, int value) => new(key, new MiniYaml(Exts.ToStringInvariant(value)));
+		internal static MiniYamlNode SaveField(string key, WVec value) => new(key, new MiniYaml(value.ToString()));
+		internal static MiniYamlNode SaveField(string key, CVec value) => new(key, new MiniYaml(value.ToString()));
```

Comment:
thid does feel a bit weird. We use FieldSaver class to handle saving arbitrary pieces of data. Use that instead.

There's also no need for saving to be of internal scope